tests/pom.xml

```diff
@@ -119,6 +119,48 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
                     </log>
                   </run>
                 </image>
+                <!-- ##### Device registry server ##### -->
+                <image>
+                  <name>eclipsehono/hono-service-device-registry</name>
+                  <build>
+                    <from>eclipsehono/hono-service-device-registry:${project.version}</from>
+                    <assembly>
+                      <mode>dir</mode>
+                      <basedir>/etc/hono</basedir>
+                      <inline>
+                        <id>config</id>
+                        <fileSets>
+                          <fileSet>
+                            <directory>${project.basedir}/src/test/resources/deviceregistry</directory>
+                            <outputDirectory>.</outputDirectory>
+                            <includes>
+                              <include>*</include>
+                            </includes>
+                          </fileSet>
+                        </fileSets>
+                      </inline>
+                    </assembly>
+                  </build>
+                  <run>
+                    <ports>
+                      <port>+deviceregistry.ip:deviceregistry.amqp.port:5672</port>
+                    </ports>
+                    <network>
+                      <mode>custom</mode>
+                      <name>hono</name>
+                      <alias>device-registry-server.hono</alias>
```

Comment:
this should match one of the *altSubjectNames* from the device registry's certificate, i.e. `device-registry.hono`